test-lib.sh

```diff
@@ -1069,4 +1054,34 @@ ct_test_app_dockerfile() {
   return $ret
 }
 
+
+# ct_run_tests_from_testset
+# -----------------------------
+# Runs all tests in $TEST_SET, prints result to
+# the $TEST_SUMMARY variable
+# Argument: app_name - application name to log
+# Uses: $TEST_SET - set of test cases to run
+# Uses: $TEST_SUMMARY - variable for storing test results
+# Uses: $IMAGE_NAME - name of the image being tested
+ct_run_tests_from_testset() {
+  local app_name="$1"
+  local testsuite_result=0
+  for test_case in $TEST_SET; do
+    info "Running test $test_case ... "
+    $test_case
+    local test_result=$?
+    local test_msg
+    local output_format="%s %s $1\n"
+    if [ $test_result -eq 0 ]; then
+      test_msg="[PASSED]"
+    else
+      test_msg="[FAILED]"
+      testsuite_result=1
```

Comment:
This could be variable like here: https://github.com/sclorg/s2i-perl-container/blob/master/test/run#L15

And then we can unify also standard docker tests.